packages/tslint-config/tslint.json

```diff
@@ -92,6 +92,7 @@
         "prefer-function-over-method": true,
         "prefer-object-spread": true,
         "prefer-readonly": true,
+        "prefer-template": true,
```

Comment:
Adding this rule applies it to the entire monorepo. I think it is a good idea but it is probably best to create a separate PR where we also go through and fix all the code that doesn't adhere to the rule.